packages/appsync-modelgen-plugin/src/__tests__/visitors/appsync-dart-visitor.test.ts

```diff
@@ -464,4 +465,25 @@ describe('AppSync Dart Visitor', () => {
       expect(generatedCode).toMatchSnapshot();
     });
   });
+
+  describe('Many To Many V2 Tests', () => {
```

Comment:
I believe this change is not specific to Dart. Can we add similar tests for JS, Swift, Java if they don't exist?

Reply:
Adding